lib/KCard/index.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+
+  <BaseCard
+    :to="to"
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :style="wrapperStyle"
+  >
+    <template #title>
+      <!-- @slot Optional slot section containing the title contents, should not contain a heading element. -->
+      <div class="title-slot">
+      <!-- <slot name="title"></slot> -->
+      </div>
+    </template>
+
+    <template #default>
+      <aside
+        v-if="thumbnailDisplay !== 'none'"
+        :style="{ dynamicOrder }"
+        :class="{
+          'image-class-vertical': layout === 'vertical',
+          'image-class-horizontal': (layout === 'horizontal' && thumbnailDisplay === 'small')
+        }"
+      >
+        <KImg
+          v-if="thumbnailSrc"
+          :src="thumbnailSrc"
+          :isDecorative="true"
+          :appearanceOverrides="{ scaleType: thumbnailScaleType, width: cardContentPartial }"
+          class="thumbnail-image"
+          :style="imageRadius"
+        />
+        <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+      </aside>
+      <div class="display-inline-block" :style="cardContentPartial">
+        <div
+          data-testid="aboveTitle"
+          class="above-title-style"
+          :class="{ 'above-title-horizontal': layout === 'horizontal' }"
+        >
+          <slot name="aboveTitle"></slot>
+        </div>
+
+        <template>
+          <div
+            :class="{ 'title-horizontal': layout === 'horizontal' }"
+            class="title-slot"
+          >
+            <slot name="title"></slot>
+          </div>
+        </template>
+
+        <div
+          data-testid="belowTitle"
+          class="below-title"
+          :class="{ 'below-title-horizontal': layout === 'horizontal' }"
+        >
+          <slot name="belowTitle"></slot>
+        </div>
+        <div
+          class="footer"
```

Comment:
I think we could add a check here, and render this node just if there is a `$slots.footer`